### PR TITLE
Set logexporter to capture konnectivity server and agent logs.

### PR DESCRIPTION
**Pull request by cheftako**
Capture konnectivity logs to help debug flaky scalability test where konnecivity-agent is sometimes restarted.

### CODE REVIEW

**Comment by liggitt** [comment]
I'm +1 on capturing these, though I don't have context to know if this change accomplishes that :)

**Comment by dims** [comment]
yes, this should do it.

/approve
/lgtm

**Comment by dims** [comment]
@cheftako some of the CI jobs pick up from different locations (the list of files to dump), may be it would be good to update all of them?
https://cs.k8s.io/?q=startupscript&i=nope&files=&excludeFiles=cluster%2Faddons&repos=kubernetes/kubernetes,kubernetes/test-infra

File: logexporter/cmd/main.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ var (
 
 	// Cloud provider specific logfiles.
 	awsLogs      = []string{"cloud-init-output"}
-	gceLogs      = []string{"startupscript"}
+	gceLogs      = []string{"startupscript", "konnectivity-*"}
```
Comment (mborsz) [inline logexporter/cmd/main.go:62]: Should it be "containers/konnectivity-*" to match https://github.com/kubernetes/test-infra/commit/8fbc6048fb3d844f8c013fb34695e307fc7124ef?

Also: https://github.com/kubernetes/test-infra/commit/8fbc6048fb3d844f8c013fb34695e307fc7124ef fetches this log only on nodes (for all cloud providers) with log-dump.sh and this change fetches only for gce and both on nodes and masters.

Is the discrepancy between log-dump.sh and logexporter intended? They both try to do the same thing so I would expect them to implement similar logic.

Comment (cheftako) [reply logexporter/cmd/main.go:62]: Working on making it better match containers. log-dump and logexporter differences were part of dims response and working on that as well. The konnectivity system is only enabled on GCE (that I know of). I intended this to be on both nodes and masters. On nodes it grabs the agent logs and on masters it should grab the server log.

**Comment by dims** [comment]
@cheftako looks like one of the CI jobs is still red. can you please assign it back to me when that is fixed? thanks!

/unassign

**Pull request closed without merging**